src/gf.c

```diff
@@ -2478,7 +2478,12 @@ static void record_precompile_statement(jl_method_instance_t *mi)
     if (!jl_has_free_typevars(mi->specTypes)) {
         jl_printf(s_precompile, "precompile(");
         jl_static_show(s_precompile, mi->specTypes);
-        jl_printf(s_precompile, ")\n");
+        if (jl_options.trace_compile_timing) {
+            jl_printf(s_precompile, ") # %.3fms\n", compilation_time / 1e6);
```

Comment:
Nit/extra-minor tweak: how about "%.3g`?

Reply:
We tend to use fixed d.p. in timing rather than s.f.  Like in `@time` and `precompile(timing=true)`

Reply:
Cool, I was just thinking might as well not show more information is helpful/reliable and that 2-3 sig fig should be plenty from that perspective.

Reply:
I think d.p. makes more sense for quick visual scanning because `12.345` looks bigger than `1.234` 
Compared to s.f. `12.3` vs. `1.23`

Reply:
Perhaps, but then you've got `ms` and `us` :shrug:.

If we're going with fixed dp though, I'd suggest that `%.1-2f` would be better that `%.3f`. It's even easier to visually scan `12.3` vs. `1.2` (given system variation, I'm not sure when 2 let alone 3 dp would actually be significant).